templates/index.jinja2

```diff
@@ -5,43 +5,17 @@
         Total pending tasks: {{ num_active_tasks }}
     </p>
 
-    {% if entries %}
-        <div class="container">
-            <div class="table-responsive">
-                <table class="table">
-                    <thead>
-                        <tr>
-                            <th scope="col">URL</th>
-                            <th scope="col">Tag</th>
-                            <th scope="col">Status</th>
-                            <th scope="col">Actions</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {% for entry in entries %}
-                            <tr>
-                                <td>{{ entry.payload.data }}</td>
-                                <td>{{ entry.payload_persistent.tag }}</td>
-                                <td>
-                                    {% if entry.num_active_tasks > 0 %}
-                                        <span class="badge bg-warning">
-                                            In progress (pending tasks: {{ entry.num_active_tasks }})
-                                        </span>
-                                    {% else %}
-                                        <span class="badge bg-success">
-                                            Done
-                                        </span>
-                                    {% endif %}
-                                </td>
-                                <td>
-                                    <a href="/analysis/{{ entry.id }}?task_filter=interesting">Show results</a>
-                                </td>
-                            </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-        </div>
+    {% if has_analyses %}
+        <table id="analysis_list" class="table table-hover">
+            <thead>
+                <tr>
+                    <th scope="col">Target</th>
+                    <th scope="col">Tag</th>
+                    <th scope="col">Status</th>
+                    <th scope="col">Actions</th>
+                </tr>
+            </thead>
+        </table>
```

Comment:
Should table render nothing only table headers?

Reply:
yup, this is filled by javascript